api/v1/endpoints/queries.sql

```diff
@@ -65,7 +65,9 @@ WITH object_data AS (
         JOIN course_instances AS ci ON (ci.id = a.course_instance_id)
         JOIN assessment_sets AS aset ON (aset.id = a.assessment_set_id)
         JOIN assessment_instances AS ai ON (ai.assessment_id = a.id)
-        JOIN users AS u ON (u.user_id = ai.user_id)
+        LEFT JOIN group_info($assessment_id) AS gi ON (gi.id = ai.group_id)
+        LEFT JOIN group_users AS gu ON (gu.group_id = gi.id)
+        JOIN users AS u ON (u.user_id = ai.user_id OR u.user_id = gu.user_id)
```

Comment:
This JOIN will result in multiple rows for the same assessment instance (one row per group user), which I don't think is what we want here (this is a query that is supposed to return one row per assessment instance). Instead I think you should do the same thing as this: https://github.com/PrairieLearn/PrairieLearn/blob/5b997eabb69a0b5707f6b0e2ab2d615069455756/pages/instructorAssessmentInstances/instructorAssessmentInstances.sql#L49

Also add new output columns for the group info, like this: https://github.com/PrairieLearn/PrairieLearn/blob/5b997eabb69a0b5707f6b0e2ab2d615069455756/pages/instructorAssessmentInstances/instructorAssessmentInstances.sql#L6